addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -172,22 +189,21 @@ class Network {
         this.sfu?.addEventListener(name, f);
     }
     /**
-     * @param {streamType} type
+     * @param {StreamType} type
      * @param {MediaStreamTrack | null} track track to be sent to the other call participants,
      * not setting it will remove the track from the server
      */
     async updateUpload(type, track) {
-        const proms = [this.p2p.updateUpload(type, track)];
-        if (this.sfu?.state === "connected") {
-            proms.push(this.sfu.updateUpload(type, track));
-        }
-        await Promise.all(proms);
+        await Promise.all([
+            this.sfu?.state === "connected" && this.sfu.updateUpload(type, track),
+            this.p2p.updateUpload(type, track),
+        ]);
     }
     /**
      * Stop or resume the consumption of tracks from the other call participants.
      *
      * @param {number} sessionId
-     * @param {Object<[streamType, boolean]>} states e.g: { audio: true, camera: false }
+     * @param {Object<[StreamType, boolean]>} states e.g: { audio: true, camera: false }
```

Comment:
This type doesn't work, it shows `any`:
<img width="524" height="132" alt="Image" src="https://github.com/user-attachments/assets/58563d65-b33b-443b-a6ee-c040d083e1ee" />